build.gradle

```diff
@@ -53,12 +51,23 @@ def dependencyVersions = [
         hk2_version           : '2.5.0-b05'
 ]
 
-group = "uk.gov.ida"
-version = "${build_version}"
-
-subprojects {
+allprojects {
+    apply plugin: 'idea'
     apply plugin: 'java'
+    apply plugin: 'com.github.ben-manes.versions'
+    group = "uk.gov.ida"
+    version = "${build_version}"
+
+    java {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Do we actually need to stipulate this?

Reply:
I don't think so, it should be enough to have just the java or source functions